src/server.js

```diff
@@ -0,0 +1,24 @@
+const {checkWatcher, startNPMWatcher} = require('./watchNPM')
+const redis = require('redis')
+const amqp = require('amqplib')
+const logger = require('~/src/logging').logger(module)
+const config = require('~/src/config')
+config.load()
+
+;(async () => {
+  const connection = await amqp.connect(config.amqUrl)
+  const channel = await connection.createChannel()
+  await channel.assertQueue(config.queueName)
+  const client = redis.createClient(config.redisURL)
```

Comment:
`config.getRedisUrl()`. `config` is a [fashion-model](https://github.com/fashion-js/fashion-model)